<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_main</class>
 <widget class="QWidget" name="widget_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grammar Correction</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="emotion_sen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:black;border:5px solid  #C9E4C2;border-radius:20;</string>
   </property>
   <property name="text">
    <string>Emotion Sensor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="g_correction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:black;border:5px solid  #C9E4C2;border-radius:20;</string>
   </property>
   <property name="text">
    <string>Grammar Correction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="w_meaning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:black;border:5px solid  #C9E4C2;border-radius:20;</string>
   </property>
   <property name="text">
    <string>Meaning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_sep">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:black;border:5px solid  #C9E4C2;border-radius:20;</string>
   </property>
   <property name="text">
    <string>Pos Separation</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="input_area">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="answer_area">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mode">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mode  Selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="answer">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:black;border:5px solid  #C9E4C2;border-radius:20;</string>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
